Clients/GUI/Scripts/NewsGetter.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using DTO;
+using DTO.RestRequests;
+using Kernel;
+using Kernel.Enums;
+
+namespace GUI.Scripts
+{
+    public static class NewsGetter
+    {
+        public static IEnumerable<NewsItem> GetNews(string feedUrl)
+        {
+            const string url = "https://localhost:5007/news/getnews";
+
+            var queryParams = new Dictionary<string, string>
+            {
+                { "feedUrl", feedUrl }
+            };
+
+            var client = new RestClient<string, IEnumerable<NewsItem>>(url, RestRequestType.GET, queryParams: queryParams);
```

Comment:
Does it work ok with deserialization of interfaces? Because when I tried to send IEnumerable of market instruments in my task, I faced some problems and switched to List, hm

Reply:
I think, it`s work well because i get news without any problems

Reply:
> I think, it`s work well because i get news without any problems

Ok I see